.gitignore

```diff
@@ -43,3 +43,4 @@ msbuild.ProjectImports.zip
 StyleCop.Cache
 UpgradeLog.htm
 .idea
+/src/Framework/App.Ref/src/PackageOverrides.txt
```

Comment:
?

Reply:
That is emitted by visual studio preview. Guess we do not need that.